projects/pedal_board/src/throttle_calib.c

```diff
@@ -0,0 +1,58 @@
+#include "throttle_calib.h"
+#include "ads1015.h"
+#include "log.h"
```

Comment:
Your throttle_calib and brake_calib are almost identical. Merge them and name it pedal_calib, and have the information with regards to pedal or brake passed in as input.